data/SivaReddy184.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "Siva Reddy",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "**Test Analyst** in profession. DevOps enthusiast &  On a learning spree",
+  "avatar": "https://avatars.githubusercontent.com/u/91452063?v=4",
```

Comment:
Please update your avatar link with a more dynamic URL. 🛠
So that it gets updated automatically when you change your profile picture!

```suggestion
  "avatar": "https://github.com/SivaReddy184.png",
```